app/views/users/show.html.erb

```diff
@@ -238,6 +238,11 @@
 <div class="richtext text-break"><%= @user.description.to_html %></div>
 
 <% if @heatmap_data.present? %>
+  <div class="row">
+    <div class="col text-body-secondary fs-5">
+      <%= t(".contributions.#{@changes_count == 1 ? 'one' : 'other'}", :changes_count => @changes_count) %>
```

Comment:
If you call the variable in the translation `count` then you won't need to work out the subkeys yourself as rails i18n will do it for you...